<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Brian Hua - Conserved epitopes</title>
<!-- Bootstrap CSS -->
<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
<link rel="stylesheet" href="../CSS/my_style.css">

</head>
<body>

<section id='projdesc' class="projdesc-section">
  <div class="container">
    <h2>Determination of highly conserved epitopes from RSV surface proteins for improved vaccine design</h2>
    <h4>Background and rationale</h4>
    <p>The immune system plays a critical role in our ability to overcome infections. Furthermore, immune memory of a previous infection can provide very effective protection against future infection. The immune system function that is involved in immune memory consists of two separate but complementary arms: the antibody arm and the T-cell arm. Recently, T-cells have become a focus on immune system-based therapeutics and preventative drugs due to the ability of T-cells to target and get rid of cells that are already infected (for example with a virus or a bacterium). To do this, T-cells recognize small fragments of proteins called epitopes that are derived from the invading organism. Thus, if we can "pre-train" T-cells to recognize specific epitopes of an infectious agent of interest, we can effectively prepare the body to quickly contain a future infection before it can develop into disease.</p>
    <p>In this example, I focused on respiratory syncytial virus (RSV), a virus that can cause severe disease especially in infants, older adults, and immuno-compromised individuals. RSV is an enveloped virus that has two protein components on the outside, the G protein that is involved in the binding of the virus and the F protein that is involved of the fusion of the virus with the host cell.</p>
    <div class="projdesc-figure">
        <img src="../images/projects/RSV_structure.jpg" class="figure-image" alt="Project 1">
        <div class="figure-body">
            <h5 class="figure-title">Schematic of the structure of respiratory syncytial virus (RSV)</h5>
            <p class="figure-text">RSV is an enveloped virus with two major proteins on its surface: the G attachment glycoprotein and the F fusion protein. FIgure from <a href="https://ijponline.biomedcentral.com/articles/10.1186/s13052-021-01148-8">Azzari et al. 2021. Italian Journal of Pediatrics.</a></p>
        </div>
    </div>
<br>
    <p>As like most viruses, there are many different strains of RSV that can infect humans, and these strains exhibit a high level of diversity. This means that viral vaccines tend to be strain-specific, and while the vaccine may provide effective protection against one strain, it may not be effective against another strain that is too genetically divergent. One strategy to overcome this challenge is to target conserved regions of RSV. In other words, if we target protein regions that are the same sequence across multiple strains of RSV, we could potentially provide broad protection across multiple strains of RSV with a single vaccine. In this project, I created a Python-based tool that takes in a set of aligned sequences from a protein of interest, divides the protein sequences up into epitope fragments, determines the conservation of each epitope across the alignment, and returns a list of the most highly conserved epitopes. These epitopes could then be used for vaccine design to improve breadth of protection.</p>
    <h4>Methods</h4>
    <p>All publicly available sequences of RSV were downloaded from the <a href="https://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information (NCBI)</a>. From these sequences, the G and F protein sequences were extracted from each sequence record using Biopython and written to separate multi-sequence fasta files. For G protein, a total of 2413 sequences were used. For F protein, a total of 2414 sequences were used. These fasta files were then used to perform a multi-sequence alignment using <a href="https://www.ebi.ac.uk/jdispatcher/msa/clustalo">Clustal Omega</a>.</p>
    <p>To evaluate the conservation of each position within the G and F proteins, each multiple-sequence alignment was parsed using the AlignIO module of Biopython. For each column of the alignment, the number of times each amino acid was in that column was counted. The count of the most common amino acid in each column was divided by the total number of sequences in the alignment to give the overall conservation at that location.</p>
    <p>To determine conserved epitopes, each multiple sequence alignment was parsed, and 8-, 9-, 10-, and 11-mers were tiled and extracted across each sequence and compiled into a single list. For each epitope in the list, the epitope presence was checked across each sequence within the alignment. Finally the epitope count was then divided by the total number of sequences in the alignment to give the epitope conservation. The histogram of the number of epitopes at each conservation level is outputted. Finally, all epitopes with conservation of greater than or equal to 0.9 are saved to a CSV as the final list of conserved epitopes.</p>
    <h4>Results</h4>
    <p>To get a sense of the overall conservation of the F and G proteins across the RSV sequences being examined, the percent conservation of each position in the G and F proteins was calculated.</p>

    <div class="projdesc-figure">
        <img src="../images/projects/F_conservation.jpg" class="figure-image" alt="Project 1">
        <img src="../images/projects/G_conservation.jpg" class="figure-image" alt="Project 1">
        <div class="figure-body">
            <h5 class="figure-title">Percent conservation at each position of the G and F proteins</h5>
            <p class="figure-text">A line plot of percent conservation of the most common amino acid at each position within the G and F proteins of RSV. 2413 and 2414 sequences were analyzed for G and F proteins, respectively.</p>
        </div>
    </div>
<br>
<p>Previous genetic diversity studies have shown that the G protein exhibits the most extensive protein sequence diversity of all the proteins of RSV (<a href="https://journals.asm.org/doi/10.1128/cmr.13.1.1">Sullender. 2000. Clin. Microbiol. Rev.</a>). From this set of sequences, it is clear that the G protein is less conserved across the protein compared to F protein. However, there are still some stretches of G protein that exhibit high conservation. To determine what these conserved epitopes are, epitope sequences from the G protein and F protein multiple sequence alignments were extracted and the conservation of each epitope was determined. The histogram of the number of epitopes for each conservation level are show below:</p>
    <div class="projdesc-figure">
        <img src="../images/projects/F_histogram.jpg" class="figure-image" alt="Project 1">
        <img src="../images/projects/G_histogram.jpg" class="figure-image" alt="Project 1">
        <div class="figure-body">
            <h5 class="figure-title">Histograms of number of epitopes at each conservation level for the RSV G and F proteins</h5>
            <p class="figure-text">8-, 9-, 10-, and 11-mer epitopes were extracted from the multiple sequence alignments for both the G and F protein. The conservation of each of these epitopes was determined, and the distribution of conservation level of the epitopes was plotted.</p>
        </div>
    </div>
    <br>  
    <p>As expected, there a lot fewer conserved epitopes in the G protein compared to the F protein. In fact, the vast majority of epitopes in the G protein are very poorly conserved compared to the F protein where there is a better distribution of conservation. Epitopes with a conservation of greater than or equal to 0.9 were then written to a CSV file for both proteins.</p>
    <div class="projdesc-figure">
        <img src="../images/projects/FG_table.jpg" class="figure-image" alt="Project 1">
        <div class="figure-body">
            <h5 class="figure-title">Representative highly-conserved epitopes from the F and G proteins of RSV</h5>
            <p class="figure-text">The first ten epitopes with conservations of 0.9 or greater are shown. A total of 1080 conserved epitopes were found for F protein, and a total of 59 conserved epitopes were found for G protein.</p>
        </div>
    </div>
    <br>
    <p>A total of 1080 conserved epitopes were found for F protein, whereas only a total of 59 conserved epitopes were found for G protein.</p>
  <div class="container">
    <h4>Summary and future directions</h4>
    <p>For both G and F proteins of RSV, this Python tool extracted candidate eptiopes based on high conservation of the epitope across thousands of RSV sequences. While the G protein was found to have a high level of protein sequence diversity overall, 59 epitopes were still identified with high levels of conservation. Whether these conserved epitopes play functional roles in the function of the protein or whether they are immunogenic need to be tested. One future feature that could be added to this tool is to show the location of the conserved epitopes relative to known functional domains on the protein. It would also be helpful to check whether any of these epitopes are already known to be immunogenic, for example by looking at the <a href="https://www.iedb.org/"> Immune Epitope Database</a>. This could place an additional layer of functional confidence of these epitopes and help establish a rank list of most promising epitopes for immunotherapy.</p>
    <p>With a set of high-confidence, highly-conserved epitopes for the F and G proteins of RSV, a therapeutic could be designed to express these epitopes in humans. The immune system could be trained to recognize these highly-conserved RSV epitopes, potentially providing robust, long-lasting, and broad protection against infection and disease by a greater number of RSV strains.</p>
    <a href="https://github.com/blhua/conserved_epitopes" class="btn btn-secondary">Source Code</a>
  </div>
</section>


<!-- Contact Me Section -->
<section id="contact" class="contact-section text-center text-white bg-dark">
  <div class="container">
    <h2>Contact Me</h2>
    <div class="row justify-content-center">
      <div class="col-lg-6">
        <p><i class="fas fa-envelope"></i><a href="mailto:brianlhua@gmail.com">brianlhua@gmail.com</a>
        <br>
        <a href="https://blhua.github.io/CV/" class="btn btn-outline-primary">My full CV</a></p>
      </div>
    </div>
    <div class="row justify-content-center">
      <div class="col-lg-2 col-md-4 mb-4">
        <a href="https://linkedin.com/in/brianlhua" target="_blank">
          <img src="../icons/linkedin.png" alt="LinkedIn" class="social-icon">
          LinkedIn
        </a>
      </div>
      <div class="col-lg-2 col-md-4 mb-4">
        <a href="https://x.com/brianlhua" target="_blank">
          <img src="../icons/X.png" alt="X" class="social-icon">
          X
        </a>
      </div>
      <div class="col-lg-2 col-md-4 mb-4">
        <a href="https://github.com/blhua" target="_blank">
          <img src="../icons/github.png" alt="GitHub" class="social-icon">
          GitHub
        </a>
      </div>
    </div>
  </div>
</section>

<!-- Sticky Navigation Bar -->
<nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
  <div class="container">
    <a class="navbar-brand" href="#home">Extraction of conserved epitopes from multiple sequence alignment</a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="navbarNav">
      <ul class="navbar-nav ml-auto">
        <li class="nav-item">
          <a class="nav-link" href="../index.html">Home</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="#contact">Contact</a>
        </li>
      </ul>
    </div>
  </div>
</nav>


<!-- Bootstrap JS and dependencies (jQuery and Popper) -->
<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.9.5/dist/umd/popper.min.js"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
</body>
</html>
