<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Brian Hua - RNA Secondary Structure</title>
<!-- Bootstrap CSS -->
<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
<link rel="stylesheet" href="../CSS/my_style.css">

  
</head>
<body>

<section id='projdesc' class="projdesc-section">
  <div class="container">
    <h2>Genome-wide analysis of RNA secondary structure in mammalian gene expression</h2>
    <h4>Background and rationale</h4>
    <p>In bacteria, synthetic transcripts with strong RNA secondary structures and a narrower range of RNA secondary structure stability around the start codon are correlated with stronger protein expression (<a href="https://www.nature.com/articles/nbt.4238">Cambray G, Guimaraes JC, and Arkin AP. Nat. Biotechnol. 2018.</a>).</p>
    <div class="projdesc-figure">
        <img src="../images/projects/bacteria_data.jpg" class="figure-image" alt="Project 1">
        <div class="figure-body">
            <h5 class="figure-title">RNA secondary structures affect protein production in <i>Escherichia coli</i></h5>
            <p class="figure-text">In a panel of 244,000 synthetic RNA sequences, it was found that more stable RNA secondary structures centered around the start codon correlated with increased protein production.</p>
        </div>
    </div>
<br>
    <p>This can provide a useful framework for enhanced expression of novel genes or gene payloads. However, it is not known if this correlation holds true in mammalian cells. Establishing this connection could provide better guidelines for transgene design and heterologous gene expression, for example in the context of gene therapy or immunotherapy in humans. Here, I calculated RNA secondary structure minimum free energy (MFE), a measure of the stability of the secondary structure, around all start codons in the human genome and compared the MFE to relative expression of the associated proteins using proteomics datasets from HEK-293 cells, a well-characterized human cell line.</p>
    <h4>Methods</h4>
    <p>RNA secondary structure MFE was calculated for three distinct regions relative to the start codon (position +1): -30:+30, +1:+60, and +31:+90. The <a href="https://www.ncbi.nlm.nih.gov/assembly/?term=NC_000001.11">human genome build 38 release 14</a> was programatically downloaded and parsed using Biopython. For each chromosome, all coding sequence (CDS) annotations were selected and the protein product name and the start codon position of each CDS was identified. From this, the sequences of the -30:+30, +1:+60, and +31:+90 regions were determined and converted to RNA sequences. Finally, data from all chromosomes were concatenated into a single table to represent the full transcriptome. Finally, MFE values were calculated for each RNA sequence using the RNAfold program from the <a href="https://www.tbi.univie.ac.at/RNA/">ViennaRNA2 package</a>. </p>
    <p>To evaluate the protein expression level of each gene, the HEK-293 human embryonic kidney cell line was chosen as a representative human cell line due to its ubiquitous use across research groups and its well-characterized cell biology. HEK-293 proteome-wide mass spectrometry made available in <a href="https://www.mcponline.org/article/S1535-9476(20)30500-4/fulltext">Geiger et al. Mol Cell. 2012.</a> was used to determine the relative expression level (intensity-based absolute quantification, or iBAQ) of each protein. The iBAQ value was averaged across technical triplicates. Next, the MFE table was joined onto the iBAQ table on the protein product. Finally, iBAQ values were sorted and analyzed globally or divided equally into ten bins to analyze correlation of MFE with expression groups ranging from low to high.</p>
    <h4>Results</h4>
    <p>First, MFE and average iBAQ values were compared globally at each of the three RNA regions. Globally, a strong linear correlation is not observed (R<sup>2</sup>=8E-5, 2E-3, and 2E-3 for -30:+30, +1:+60, and +31:+90 regions, respectively).</p>

    <div class="projdesc-figure">
        <img src="../images/projects/STR3030_scatter.jpg" class="figure-image" alt="Project 1">
        <img src="../images/projects/STR160_scatter.jpg" class="figure-image" alt="Project 1">
        <img src="../images/projects/STR3090_scatter.jpg" class="figure-image" alt="Project 1">
        <div class="figure-body">
            <h5 class="figure-title">Protein expression and RNA secondary structures at the -30:+30, +1:+60, and +31:+90 regions around the start codon</h5>
            <p class="figure-text">A scatter plot of average iBAQ value (higher iBAQ represents higher protein expression) vs. MFE (lower MFE represents a more stable RNA secondary structure). Linear correlation was assessed via Pearson correlation coefficient. R<sup>2</sup>=8E-5, 2E-3, and 2E-3 for -30:+30, +1:+60, and +31:+90 regions, respectively.</p>
        </div>
    </div>
<br>
<p>As linear correlations between iBAQ and MFE values were not observed globally, all proteins were then grouped into 10 equally-sized expression groups based on iBAQ score. Kernel density estimate plots were then constructed for the low-, middle-, and high-expression iBAQ groups</p>   
    <div class="projdesc-figure">
        <img src="../images/projects/STR3030.jpg" class="figure-image" alt="Project 1">
        <img src="../images/projects/STR160.jpg" class="figure-image" alt="Project 1">
        <img src="../images/projects/STR3190.jpg" class="figure-image" alt="Project 1">     
        <div class="figure-body">
            <h5 class="figure-title">Protein expression groups and RNA secondary structures around the start codon</h5>
            <p class="figure-text">KDE plots of the low-, middle-, and high-expression iBAQ groups at the -30:+30, +1:+60, and +31:+90 regions around the start codon. Group 1, group 5, and group 10 represent the high-, medium-, and low-expression groups, respectively.</p>
        </div>
    </div>
    <br>  
    <p>Interestingly, at the -30:+30 and +1:+60 regions, the high-expression groups tend to have lower MFE values (stronger RNA secondary structures). This suggests that near the start codon, stronger RNA secondary structure correlates with higher protein expression of that gene. This correlation is not observed further downstream of the start codon at the +31:+90 region. Consistent with trends reported in bacteria, RNA secondary structures containing and proximal to the start codon may play important roles in gene expression.</p>
  <div class="container">
    <h4>Summary and future directions</h4>
    <p>Global analysis of RNA secondary structure reveals that stronger RNA secondary structure at or promixal to the start codon is correlated with higher gene expression in HEK-293 cells. This is a signficant finding as it can help establish better guidelines for transgene design for optimal protein expression. For example, increased heterologous expression of a target protein in a human gene therapy or vaccine therapeutic may be achieved through codon-optimization approaches that prioritize stronger transcript secondary structures at the start codon.</p>
    <p>The biological reason as to why stronger transcript secondary structures at the start codon leads to higher protein expression remains unclear. One hypothesis is that stronger secondary structure at the start codon stablizes the transcript, effectively extending the half-life of the transcript in the cell for prolonged translation capability. Another hypothesis is that a strong secondary structure at the start codon helps regulate translation through ribosome processivity, resulting in more stable translation complexes and ultimately a higher inherent success rate of full protein synthesis. Future molecular biology studies are needed to test these hypotheses.</p>
    <p>Looking forward, it is imperative to extend this current analysis to additional datasets. This includes other proteomics data sets for HEK-293 cells and, more importantly, other human cell lines and tissue types. This will shed more light on whether this correlation is cell line-/cell type-specific or if it is a more global mechanism underlying protein expression in mammalian cells. Finally, it would be informative to test candidate transgenes side-by-side that are engineered to have different levels of RNA secondary structure while still encoding for the same protein.</p>
    <a href="https://github.com/blhua/MFE_expression" class="btn btn-secondary">Source Code</a>
    <br>
    <br>
    <a href="../index.html" class="btn btn-primary">Home</a>
    
  </div>
</section>


<!-- Contact Me Section -->
<section id="contact" class="contact-section text-center text-white bg-dark">
  <div class="container">
    <h2>Contact Me</h2>
    <div class="row justify-content-center">
      <div class="col-lg-6">
        <p><i class="fas fa-envelope"></i><a href="mailto:brianlhua@gmail.com">brianlhua@gmail.com</a>
        <br>
        <a href="https://blhua.github.io/CV/" class="btn btn-primary">My full CV</a></p>
      </div>
    </div>
    <div class="row justify-content-center">
      <div class="col-lg-2 col-md-4 mb-4">
        <a href="https://linkedin.com/in/brianlhua" target="_blank">
          <img src="../icons/linkedin.png" alt="LinkedIn" class="social-icon">
          LinkedIn
        </a>
      </div>
      <div class="col-lg-2 col-md-4 mb-4">
        <a href="https://x.com/brianlhua" target="_blank">
          <img src="../icons/X.png" alt="X" class="social-icon">
          X
        </a>
      </div>
      <div class="col-lg-2 col-md-4 mb-4">
        <a href="https://github.com/blhua" target="_blank">
          <img src="../icons/github.png" alt="GitHub" class="social-icon">
          GitHub
        </a>
      </div>
    </div>
  </div>
</section>

<!-- Sticky Navigation Bar -->
<nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
  <div class="container">
    <a class="navbar-brand" href="#home">RNA secondary structure</a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="navbarNav">
      <ul class="navbar-nav ml-auto">
        <li class="nav-item">
          <a class="nav-link" href="../index.html">Home</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="#contact">Contact</a>
        </li>
      </ul>
    </div>
  </div>
</nav>


<!-- Bootstrap JS and dependencies (jQuery and Popper) -->
<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.9.5/dist/umd/popper.min.js"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
</body>
</html>
